vcpkg/concepts/ports.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ports
+description: Ports as a vcpkg concept
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 09/21/2023
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+
+# Concept: Ports
+
+A port in vcpkg is a versioned recipe that produces a set of files. It also imposes constraints on the install graph, ensuring compatible dependency resolution. At a minimum, a port contains:
```

Comment:
```suggestion
A port in vcpkg is a versioned recipe that produces a set of files. Executing a port affects the installation graph by producing new header or binary files. Ports may have dependencies on other ports and may require additional dependencies when certain features are enabled, these dependencies must be reflected in the ports metadata. At a minimum, a port contains:
```
The original sentence is not clear to me, are we trying to say that ports install things? that ports may depend on other ports? or both?